app/models/data.py

```diff
@@ -1,22 +1,43 @@
-from sqlalchemy import Column, Integer, Date, Time, String, Float
+from sqlalchemy import Column, Integer, Date, Time, String, Float, ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
 
 from db.base_class import Base
 
 
 class Data(Base):
     __tablename__ = 'data'
     id = Column(Integer, primary_key=True, unique=True, nullable=False)
+
     date = Column(Date, nullable=True)
     time = Column(Time, nullable=True)
-    nuts3 = Column(String, nullable=True)
-    nuts2 = Column(String, nullable=True)
-    temperature_air = Column(Float, nullable=True, info={"unit_of_measure": "celsius"})
-    relative_humidity = Column(Float, nullable=True, info={"unit_of_measure": "percentage"})
+    parcel_location = Column(String, nullable=True)
+
+    atmospheric_temperature = Column(Float, nullable=True, info={"unit_of_measure": "celsius"})
+    atmospheric_temperature_daily_min = Column(Float, nullable=True, info={"unit_of_measure": "celsius"})
+    atmospheric_temperature_daily_max = Column(Float, nullable=True, info={"unit_of_measure": "celsius"})
+    atmospheric_temperature_daily_average = Column(Float, nullable=True, info={"unit_of_measure": "celsius"})
+    atmospheric_relative_humidity = Column(Integer, nullable=True, info={"unit_of_measure": "percentage"})
```

Comment:
can't this value be float?

Reply:
Yes, and I thought I changed it, since the schemas and APIs work with floats instead of integers, changing to float in model and db migration.